README_AWS.md

```diff
@@ -40,7 +40,7 @@ Set up your environment to use the correct credentials for the AWS account for t
 Ensure that the IAM support role policy (default: ManagedOpenShift-Support-Role-Policy) includes the following permissions.
 ```json
 {
-  "Version": "2012-10-17",
+  "Version": "<version>",
```

Comment:
This is a specific required string from AWS
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_version.html

Reply:
done